### PR TITLE
Remove section 5

### PR DESCRIPTION
**Pull request by svenvanderburg**

### CODE REVIEW

**Review by carschno — APPROVED** [review]

File: fig/course-overview.svg
Comment (sjvrijn) [inline fig/course-overview.svg:1]: The font size seems to have gotten smaller, and there are string quotes `''` around the labels. Can those be removed?

File: _episodes/00-setting-the-scene.md [hunk 1]
```diff
@@ -57,23 +57,18 @@ are transferable to other similar tools and programming languages.
...
---> E(5. Managing software
-over its lifetime)
+A[1. Setting up
+software environment]
+--> B[2. Verifying
+software correctness]
+--> C[3. Software development
+as a process]
+--> D[4. Collaborative
+development for reuse]
```
Comment (sjvrijn) [inline _episodes/00-setting-the-scene.md:71]: Out of curiosity: why the change from `()` to `[]`?

File: fig/course-overview.svg
Comment (svenvanderburg) [reply fig/course-overview.svg:1]: Yeah, probably yes. But it took me longer than I wanted it to take to adapt this figure 😂 So I am actually quite happy with and proud of the result. Of course if this would be used by the rest of the world and not just this workshop we should make it prettier, but in this case I am happy with it.

File: _episodes/00-setting-the-scene.md (hunk 1 above)
Comment (svenvanderburg) [reply _episodes/00-setting-the-scene.md:71]: The way it was it didn't work. And this works but it creates the ticks. I am content with not fully understanding at this moment 🙈😬

Comment (sjvrijn) [reply _episodes/00-setting-the-scene.md:71]: Fair enough 🙂

**Comment by carschno** [comment]
For some reason, I still see the old image (including section 5) here: https://esciencecenter-digital-skills.github.io/python-intermediate-development-gitlab/40-section4-intro/index.html
@svenvanderburg @sjvrijn any clues?

**Comment by svenvanderburg** [comment]
@carschno yes we completely missed it... oops. I fixed it with a cowboy commit 🤠 oops https://github.com/esciencecenter-digital-skills/python-intermediate-development-gitlab/commit/af4f745e7db24b416a3ac8244f8ff5c531c9e0d8
